package.json

```diff
@@ -3,7 +3,9 @@
   "version": "1.0.0",
   "description": "Web API Challenge",
   "main": "index.js",
-  "scripts": {},
+  "scripts": {
+    "server": "nodemon index.js"
+  },
```

Comment:
dont forget the start script to avoid having to add it when it comes time to eventually deploy